RELEASING.md

```diff
@@ -19,11 +36,31 @@ development iteration.
 
 The resources that require localization (e.g. `Messages.properties`, `vulnerabilities.xml`) are uploaded to the OWASP ZAP projects in
 [Crowdin](https://crowdin.com/) when the main release is released, if required (for pre-translation) the resources can be uploaded manually
-at anytime by running the workflow [Crowdin Upload Files](https://github.com/zaproxy/zaproxy/actions/workflows/crowdin-upload-files.yml).
+at any time by running the workflow [Crowdin Upload Files](https://github.com/zaproxy/zaproxy/actions/workflows/crowdin-upload-files.yml).
 
 The resulting localized resources are added/updated in the repository periodically (through a workflow in the
 [zap-admin repository](https://github.com/zaproxy/zap-admin/)).
 
+### Post Release
+
+- [ ] Publish blog post
+- [ ] Announce on
+  - [ ] ZAP User and Dev groups
+  - [ ] @zaproxy twitter account
+  - [ ] OWASP Slack
+- [ ] Release client APIs
+  - [ ] [Java](https://github.com/zaproxy/zap-api-java/blob/main/RELEASING.md)
+  - [ ] [Python](https://github.com/zaproxy/zap-api-python/blob/master/RELEASING.md)
+- [ ] Update major projects using ZAP
+  - [ ] Kali - [new issue](https://bugs.kali.org/)
+  - [ ] Flathub
```

Comment:
Add link? https://github.com/flathub/org.zaproxy.ZAP

Reply:
I think this one we just have to manually update things. I couldn't find any defined/automated process.